api/queries_issue.go

```diff
@@ -78,7 +78,7 @@ const fragments = `
 		url
 		state
 		updatedAt
-		labels(first: 3) {
+		labels(first: 100) {
```

Comment:
I think 100 labels is a bit excessive, what about setting this to 25 for now?

Reply:
I think 100 is same as saying fetch all labels in GitHubs graphql api. I wouldn't expect labels to be that large anyways, and I noticed use of 100 in other parts of code querying labels. Better to keep at 100